jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -127,13 +127,23 @@ data class CodeModernizerSessionContext(
                         dependencyfiles.forEach { depfile ->
                             val relativePath = File(depfile.path).relativeTo(depDirectory.parentFile)
                             val paddedPath = depSources.resolve(relativePath)
-                            it.putNextEntry(paddedPath.toPath().toString(), depfile.inputStream())
+                            var paddedPathString = paddedPath.toPath().toString()
+                            // Convert Windows file path to work on Linux
+                            if (File.separatorChar != '/') {
+                                paddedPathString = paddedPathString.replace("\\", "/")
+                            }
+                            it.putNextEntry(paddedPathString, depfile.inputStream())
                         }
                     }
                     files.forEach { file ->
                         val relativePath = File(file.path).relativeTo(sourceFolder)
                         val paddedPath = zipSources.resolve(relativePath)
-                        it.putNextEntry(paddedPath.toPath().toString(), file.inputStream)
+                        var paddedPathString = paddedPath.toPath().toString()
+                        // Convert Windows file path to work on Linux
+                        if (File.separatorChar != '/') {
+                            paddedPathString = paddedPathString.replace("\\", "/")
+                        }
+                        it.putNextEntry(paddedPathString, file.inputStream)
```

Comment:
`file.inputStream` here versus `depfile.inputStream()` above?

Reply:
It's the existing implementation, and I assume it's because `file` is type `VirtualFile`, while `depfile` is type `File`, so method signatures are different.